site/docs/pages/wallet/types.mdx

```diff
@@ -74,6 +74,21 @@ export type WalletDropdownDisconnectReact = {
 };
 ```
 
+## `WalletDropdownFundLinkReact`
+
+```ts
+export type WalletDropdownFundLinkReact = {
+  className?: string; // Optional className override for the element
+  href: string;
+  icon?: ReactNode;
+  rel?: string;
+  openIn?: 'window' | 'tab'; // Whether to open the funding flow in a tab or a window
```

Comment:
is this is a `popup` should we probably use that word instead?

Reply:
it is a **popup window** but I like `window` terminology better since `popup` has a negative connotation and makes it seems like it's spam (popup spam)

Reply:
`window` is a very special word in web. As it means both the global object, and the window browser size.

I am kind of hesitant to use that word.